pkgs/by-name/pr/procodile/package.nix

```diff
@@ -20,6 +20,6 @@ bundlerApp {
       nicknovitski
     ];
     platforms = platforms.unix;
-    mainProgram = "procodile";
+    mainProgram = pname;
```

Comment:
please revert this change, it is considered an anti pattern to use pname due to it being possible to override